testnets/symphonytestnet3/chain.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "symphonytestnet3",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Symphony Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "symphony-testnet-3",
+  "bech32_prefix": "symphony",
+  "daemon_name": "symphonyd",
+  "node_home": "$HOME/.symphonyd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "note",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "note"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Orchestra-Labs/symphony",
+    "recommended_version": "v23.0.5",
```

Comment:
The current version is named "v23.0.5" here, but as "v23.0.5-rc1" in the versions file.